src/Cedar/IPC.c

```diff
@@ -1684,7 +1684,21 @@ void IPCSendIPv4(IPC *ipc, void *data, UINT size)
 		else if (ipv4[0] >= 224 && ipv4[0] <= 239)
 		{
 			// IPv4 Multicast
-			is_broadcast = true;
+			UCHAR dest[6];
+
+			// Per RFC 1112, multicast MAC address has the form 01-00-5E-00-00-00,
+			// where the lowest 23 bits are copied from the destination IP address.
+			dest[0] = 1;
+			dest[1] = 0;
+			dest[2] = 0x5e;
+			dest[3] = 0x7f & ipv4[1];
+			dest[4] = ipv4[2];
+			dest[5] = ipv4[3];

```

Comment:
Shouldn't this be

```suggestion
			dest[3] = 0x7f & ipv4[0];
			dest[4] = ipv4[1];
			dest[5] = ipv4[2];

```

?

From https://tools.ietf.org/html/rfc1112#section-6.4:
> An IP host group address is mapped to an Ethernet multicast address
  by placing the low-order 23-bits of the IP address into the low-order
  23 bits of the Ethernet multicast address 01-00-5E-00-00-00 (hex).

Reply:
Network byte is HIGH-low. So a low-order bit is the right-most bit.
Take the popular mDNS as an example.
```
IPv4 destination: 224.  0.  0.251
                        ↓   ↓   ↓
Mapped to: 01: 00: 5e: 00: 00: fb
```
![image](https://user-images.githubusercontent.com/54519668/115942836-b6b5e100-a4de-11eb-9268-b3d2969aa9db.png)
(Sender is an iPhone)

Reply:
You're right, thanks!